Tests/ParsingTests.swift

```diff
@@ -380,4 +380,99 @@ final class ParsingTests: XCTestCase {
         XCTAssertEqual(result.errors[0].line, 15)
         XCTAssertEqual(result.errors[0].message, "use of undeclared identifier 'unknown'")
     }
+
+    // MARK: - Executable Parsing Tests
```

Comment:
Consider adding a test for TOON format output with executables, similar to existing TOON tests in TOONEncodingTests.swift.